Makefile

```diff
@@ -7,6 +7,7 @@ help:
 	@echo "make dev               Run the app in the development server"
 	@echo "make lint              Run the code linter(s) and print any warnings"
 	@echo "make test              Run the unit tests"
+	@echo "make debug             Run the unit tests in interactive mode on http://localhost:9877"
```

Comment:
The default port is **9876**. Karma will however automatically increment this if this is already in use. I would suggest leaving the port out here. Karma prints the chosen port number to the console when it runs.

Reply:
I don't think that `make debug` makes it super clear what the difference is from `make test`. The main difference is that `make test` runs the tests once and then exits. `gulp test-watch` runs the tests once and then starts watching for changes to source files and re-runs whenever they change, or whenever a new browser connects. The test server is still run on localhost:9876 in both cases, its just that when running the tests once, it is hard to connect a browser after they start.

Perhaps something like this:

```
make test - Run the tests once
make servetests - Start the test server
```

It might also be worthwhile adding a note in the development docs (in docs/developers/developing.rst) about how to use an interactive debugger with tests.